website/integrations: add account linking note for WriteFreely (#5804)
goauthentik/authentik: website/integrations/services/writefreely/index.md

Restart writefreely.service

## Account linking

> If your username in authentik and your WriteFreely username differ, you may need to link your accounts before being able to use SSO.

Reviewer: ```suggestion
If your usernames in authentik and WriteFreely are different, you might need to link your accounts before being able to use SSO.
```

> This can be done after logging into Writefreely with local credentials, navigating to "Customize" in the menu and "Account settings" afterwards. Amongst the option "Linked Accounts", click on "authentik", and your accounts will be linked.

Reviewer: ```suggestion
To link the accounts, first log into Writefreely with local credentials, and then navigate to **Customize -->Account Settings**. In the option "Linked Accounts", click on "authentik".
```
